src/css/styles.css

```diff
@@ -151,16 +151,14 @@
   left: -60%;
   margin: 0 auto;
   text-align: center;
-  min-width: 100px;
   padding: 5px;
   background-color: var(--tooltip-bg);
   box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.2);
   border-radius: 4px;
   z-index: 99;
   display: block;
-  transition: all .4s;
+  transition: all 0.4s;
   transform: translateY(calc(-100% - 15px));
```

Comment:
can you explain why subtract `15px`   ? maybe leave a small comment in there.

Reply:
The transform with the -15px reduction is not part of this PR. I had a look at its implementation - it is to reposition the tooltip vertically to match the slider tooltip. But I think this is a bit off.
<img width="788" alt="Screenshot 2019-10-20 at 21 56 08" src="https://user-images.githubusercontent.com/20427418/67166157-c18d3400-f384-11e9-87d5-0316e9c6b305.png">
notice how there is a bit more space at the top of the non slider tooltip. Also the slider tooltip has a triangle indicating the direction the tooltip is spawning from. Is there a ticket open to use only one tooltip style across the toolbar?

Reply:
`translateY(calc(-100% - 11px));` or `translateY(calc(-100% - 0.6875rem));`
Fixes the disparity in height, but there is still a difference in styles between the two tooltips. 
<img width="774" alt="Screenshot 2019-10-20 at 22 02 14" src="https://user-images.githubusercontent.com/20427418/67166282-e209be00-f385-11e9-9289-db83e7130006.png">

Will update the PR with the refactored value and leave a comment to detail the logic.

Reply:
Oh, not yet, feel free to open it, or any issue you encounter, thanks again :v:

Reply:
I agree it should be consistent :+1:

Reply:
Oh sorry that line is probably my fault :stuck_out_tongue:

Reply:
I guess we need a smart position logic for `tooltips`, ideally they should be at the center until there is no more space the it should align to right / left.